prow/cluster/starter.yaml

```diff
@@ -0,0 +1,247 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: plugins
+data:
+  plugins:
```

Comment:
Do you want to add any plugins here?

Reply:
heart, yuks and shrug for sure

Reply:
`starter.yaml` is totally repo-unaware in this PR. There is a TODO at the bottom to show how to add plugins and the like.